proto/axelar/evm/v1beta1/query.proto

```diff
@@ -169,12 +169,18 @@ enum TokenType {
   TOKEN_TYPE_EXTERNAL = 2 [ (gogoproto.enumvalue_customname) = "External" ];
 }
 
+// ERC20TokensRequest describes the chain for which the type of ERC20 tokens are
+// requested.
 message ERC20TokensRequest {
   string chain = 1;
   TokenType type = 2;
 }
 
-message ERC20TokensResponse { repeated string assets = 1; }
+// ERC20TokensResponse describes the asset and corresponding symbol for all
+// ERC20 tokens requested for a chain
+message ERC20TokensResponse {
+  map<string, string> tokens = 1 [ (gogoproto.nullable) = false ];
```

Comment:
Another option I considered was to pass in a denom type in the request to either return a list of assets or symbols, but decided that this would be more useful to easily map between the two as well.

Reply:
Since applications aren't using this query yet, and I plan to apply this as a patch to v0.19, I didn't keep the response type backwards compatible. Let me know if you prefer otherwise.

Reply:
I don't like this mapping from asset to symbol. It's not suggested by the name of the query by any means. I'd prefer an array of { asset, symbol }.